docs/core/compatibility/containers/8.0/aspnet-port.md

```diff
@@ -85,6 +87,8 @@ There are two ways to respond to this breaking change:
 - Recommended: Explicitly set the `ASPNETCORE_HTTP_PORTS`, `ASPNETCORE_HTTPS_PORTS`, and `ASPNETCORE_URLS` environment variables to the desired port. Example: `docker run --rm -it -p 9999:80 -e ASPNETCORE_HTTP_PORTS=80 <my-app>`
 - Update existing commands and configuration that rely on the expected default port of port 80 to reference port 8080 instead. Example: `docker run --rm -it -p 9999:8080 <my-app>`
 
+If your app was built using the older <xref:Microsoft.AspNetCore.WebHost.CreateDefaultBuilder?displayProperty=nameWithType> method, either set `ASPNETCORE_URLS` (not `ASPNETCORE_HTTP_PORTS`) or update the expected default port of port 80 to reference port 5000 instead.
```

Comment:
```suggestion
If your app was built using the older <xref:Microsoft.AspNetCore.WebHost.CreateDefaultBuilder?displayProperty=nameWithType> method, set `ASPNETCORE_URLS` (not `ASPNETCORE_HTTP_PORTS`) - updating the expected default port won't work since port 5000 is not accessible outside the container.
```

Reply:
And this @tmds?

Reply:
> If your app was built using the older WebHost API, set the `ASPNETCORE_URLS` environment variable to configure the port to expose. For example, to expose port 8080 set `ASPNETCORE_URLS` to `http://*:8080`.